clients/admin-ui/src/features/system/add-multiple-systems/AddMultipleSystems.tsx

```diff
@@ -246,6 +260,14 @@ export const AddMultipleSystems = ({ redirectRoute }: Props) => {
     .getSelectedRowModel()
     .rows.filter((r) => !r.original.linked_system).length;
 
+  const totalFilters =
+    tableInstance.getState().columnFilters.length > 0
+      ? // @ts-ignore
```

Comment:
For some reason TS thinks the `tableInstance` below is `unknown`. I wasn't able to make it happy